deployment/dobi.yaml

```diff
@@ -40,16 +40,21 @@ job=deploy-eventhubdb:
              kubectl create namespace $TENANT --dry-run=client -o yaml | kubectl apply -f - &&
              kubectl delete secret ci4rail-harbor-secret --namespace=$TENANT --ignore-not-found &&
```

Comment:
This is valid for all env variable accesses:
Always prefer this syntax `${VAR}` over `$VAR`.
Here is why:
```
foo="foo"
foobar="hello world"
echo $foobar // outputs `hello world`
echo ${foo}bar //outputs `foobar`
```